server/src/main/scala/com/cloudera/livy/utils/LineBufferedStream.scala

```diff
@@ -39,7 +39,7 @@ class LineBufferedStream(inputStream: InputStream) extends Logging {
       for (line <- lines) {
         _lock.lock()
         try {
-          trace("stdout: ", line)
+          info("stdout: ", line)
```

Comment:
Can we copy this `line` and make the `info` out of lock. My concern is that log is too heavy inside the lock.